scripts/fbi/hate_crime/geo_id_resolver.py

```diff
@@ -0,0 +1,270 @@
+    'cheektowaga town ny': '3615000',
+    'camden county police department nj': '3410000',
+    'waterford township mi': '2612584240',
+    'southampton town ny': '3668462',
+    'methuen ma': '2540710',
+    'irondequoit town ny': '3637737',
+    'west seneca town ny': '3680907',
+    'webster town and village ny': '3678960',
+    'pocono mountain regional pa': '4251912',
+    'braintree ma': '2507740',
+    'lancaster town ny': '3641135',
+    'grand blanc township mi': '2633300',
+    'brighton town ny': '3608257',
+    'watertown ma': '2573440',
+    'austintown oh': '3903184',
+    'brownstown township mi': '2611220',
+    'newburgh town ny': '3650045',
+    'flint township mi': '2629020',
+    'rotterdam town ny': '3663924',
+    'los alamos nm': '3542320',
+    'big bear ca': '0606406',
+    'spring valley tx': '4869830',
+    'washington dc': '1150000',
+    'boise id': '1608830',
+    'west valley ut': '4983470',
+    'amherst town ny': '3602902000',
+    'ventura ca': '0665042',
+    'ramapo town ny': '3608760510',
+    'canton township mi': '2613120',
+    'clarkstown town ny': '3608715968',
+    'west bloomfield township mi': '2612585480',
+    'weymouth ma': '2578972',
+    'hempstead village ny': '3633139',
+    'irvington nj': '3401334450',
+    'bloomfield nj': '3401306260',
+    'west orange nj': '3401379800',
+    'redford township mi': '2616367625',
+    'greenburgh town ny': '3611930367',
+    'barnstable ma': '2503690',
+    'freeport village ny': '3627485',
+    'de kalb il': '1719161',
+    'meridian township mi': '2606553140',
+    'greenacres city fl': '1227322',
+    'saginaw township mi': '2614570540',
+    'pittsfield township mi': '2616164560',
+    'montclair nj': '3401347500',
+    'commerce township mi': '2612517640',
+    'orangetown town ny': '3608755211',
+    'haverstraw town ny': '3608732765',
+    'yorktown town ny': '3611984077',
+    'blackman township mi': '2607508760',
+    'independence township mi': '2612540400',
+    'belleville nj': '3401304695',
+    'orion township mi': '2612561100',
+    'bethlehem town ny': '3600106354',
+    'carmel town ny': '3607912529',
+    'guilderland town ny': '3600131104',
+    'houma la': '2236255',
+    'franklin ma': '2525172',
+    'spring valley village ny': '3670420',
+    'mount lebanon pa': '4200351696',
+    'paso robles ca': '0622300',
+    'white lake township mi': '2612586860',
+    'orange city nj': '3401313045',
+    'orchard park town ny': '3602955277',
+    'port chester village ny': '3659223',
+    'la canada flintridge ca': '0639003',
+    'espanola nm': '3525170',
+    'kensington ca': '0638086',
+    'sansom park village tx': '4865660',
+    'broadmoor ca': '0608338',
+    'carmel ca': '0611250',
+    'angels camp ca': '0602112',
+    'port mansfield tx': '4858928'
+}
+
+with open(os.path.join(_CITY_GEOCODES_PATH), encoding='utf8') as csvfile:
+    csv_reader = csv.reader(csvfile)
+    for a in csv_reader:
+        _CITY[a[0]] = a[1]
```

Comment:
should the dcid manually initialized above be overridden if it exists in the csv?

Reply:
The expectation is that the manually initialized map and csv do not overlap.
Have added an assertion to ensure that the keys are not present at both places.

Reply:
lgtm